distributedlog-core/src/main/java/com/twitter/distributedlog/BKSyncLogReaderDLSN.java

```diff
@@ -116,7 +110,7 @@ public void resumeReadAhead() {
 
     @Override
     public void onSuccess(LogRecordWithDLSN record) {
-        setLastSeenDLSN(record.getDlsn());
+        this.lastSeenDLSN = record.getDlsn();
```

Comment:
@leighst I pushed another change to remove 'synchronized' block. please review it again. thank you in advance.